api/ofdpa.proto

```diff
@@ -65,6 +65,12 @@ service OfdpaRpc {
 
   rpc ofdpaStgStatePortSet(openconfig_spanning_tree.Stp_Rstp_Interface_State) returns (OfdpaStatus) {}
 
+  rpc ofdpaTrunkCreate(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+  rpc ofdpaTrunkDelete(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
```

Comment:
e.g. this should be just
```
rpc ofdpaTrunkDelete(portNum) returns (OfdpaStatus) {}
```

as that is all it takes (and matches other ofdpa*Delete calls)